finetune/base_models/bert/model.py

```diff
@@ -170,6 +170,23 @@ def get_encoder(cls, config=None, **kwargs):
             return RoBERTaEncoder(**kwargs)
 
 
+class RoBERTaPos(RoBERTa):
+    positional_info = True
+    # TODO: ideally we would pull out the positional logic so that
+    # it can be easily incorporated throughout all the SourceModels
+    featurizer = bert_featurizer_with_pos
+
+    @classmethod
```

Comment:
Is there a reason these can not be inherited from RoBERTa, I don't see a difference?